packages/@okta/vuepress-site/guides/sign-into-spa/section03/react/config.md

```diff
@@ -1,2 +1,71 @@
-In your application's `App.js` file, import the following objects and pass in your configuration:
-{code}
\ No newline at end of file
+Configure a Security component in your top-level component, 'App.jsx'. The Security component will create a child context which exposes the Auth object to child components.
```

Comment:
I absolutely adore and support the use of `App.jsx` (seriously, you have no idea)....however create-react-app DOES now default to `App.js`, so can we mention both?